azure-pipelines.yml

```
@@ -35,6 +35,7 @@ jobs:
       yapf --version
       isort --version
       clang-format -version
+      echo $PATH
```

Since we don't output the commands, it's not quite clear what the output will be representing, so something like this could help:

```suggestion
      echo "PATH=$PATH"
```